examples/prom/dg_advection_global_rom.cpp

```diff
@@ -948,6 +952,23 @@ int main(int argc, char *argv[])
         osol.close();
     }
 
+    // 13. print timing info
+    if (myid == 0)
+    {
+        if (fom || offline)
```

Comment:
Some duplication could be eliminated by defining
std::string mode = (fom || offline) ? "FOM" : "ROM";

Reply:
I guess I'll leave it this time since there are only two occasions that do ``fom || online`` (or the other way round).